spec/griddler/email_spec.rb

```diff
@@ -476,6 +476,15 @@ def email_with_params(params)
     expect(headers[header_name]).to eq "invalid utf-8 bytes are ÀÁõúþÿ."
   end
 
+  it 'deeply cleans invalid UTF-8 bytes from a hash when it is submitted' do
+    header_name = 'Arbitrary-Header'
+    header_value = "invalid utf-8 bytes are \xc0\xc1\xf5\xfa\xfe\xff."
+    header = { header_name => { "a" => [header_value] } }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.